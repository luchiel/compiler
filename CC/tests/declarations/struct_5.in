struct a
{
    struct b u;
    struct b { int u; } f;
} c;