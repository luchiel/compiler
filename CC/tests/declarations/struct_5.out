type <int>
type <float>
type <void>
type <int*>
	pointer to <int>
type <void*>
	pointer to <void>
variable NULL of type <void*> initialized by
	{int 0}
		type <int>
type <0>
	struct 0
		tag a
		type <1>
			struct 1
				tag b
				variable u of type <int>
		variable u of type <1>
		variable f of type <1>
variable c of type <0>
